docs/develop/guides/cheat-sheets/momento-cache-python-cheat-sheet.md

```diff
@@ -0,0 +1,115 @@
+In this example, we use the client function above to get a client object and then use it to list all of the cache's for this account.
+```python
+  def list_caches() -> None:
+      print("Listing caches:")
+      with MomentoBasics.client() as client:
+        list_caches_response = client.list_caches()
+        match list_caches_response:
+            case ListCaches.Success() as success:
+                for cache_info in success.caches:
+                    print(f"- {cache_info.name!r}")
+            case ListCaches.Error() as error:
+                print(f"Error creating cache: {error.message}")
+            case _:
+                print("Unreachable")
+        print("")
+```
+## Write an item to a cache
+A simple example of doing a set operation. In the client.set call, you could pass a fourth value, the TTL and this would override the default TTL value set with the client connection object.
+```python
+  def set(cache_name, key, value, ttl=None):
+    with MomentoBasics.client() as client:
+      resp = client.set(cache_name, key, value, ttl)
+      match resp:
+        case CacheSet.Success():
+            pass
+        case CacheSet.Error() as error:
+            print(f"Error incrementing value: {error.message}")
+        case _:
+            print("Unreachable")
+
+## Read an item from a cache
+This is an example of a simple read operation to get an item from a cache.
+```python
+  def get(cache_name, key):
+    with MomentoBasics.client() as client:
+      resp = client.get(cache_name, key)
+      if isinstance(resp, CacheGet.Hit):
+        return resp.value_string
+      elif isinstance(resp, CacheGet.Miss):
+        raise ValueError("There is a cache miss. That key does not exist in this cache.", cache_name, "key_name : " + key)
+      elif isinstance(resp, CacheGet.Error):
+        raise resp.inner_exception
+```
+## Increment the value of an item in a cache
+An example of incrementing a value in a key. You can pass in any number
+```python
+  def incr(cache_name, key, value:int = 1):
+    with MomentoBasics.client() as client:
+      resp = client.increment(cache_name, key, value)
+      match resp:
+        case CacheIncrement.Success():
+            pass
+        case CacheIncrement.Error() as error:
+            print(f"Error incrementing value: {error.message}")
+        case _:
+            print("Unreachable")
+```
+
+Click [here](../../../../static/code/cheat-sheets/MomentoBasics.py) to see the class file with all definitions in it.
+
+## Notes of further usage
+It could be adventageous to reuse the CacheClient connection object across multiple API calls depending on how your app works. If this is something you want to do, then you should modify the client() function definition to return a CacheClient object like so:
+```python
+def client() -> CacheClient:
+```
+
+The modify the other function definitions to accept a CacheClient object as an argument, but also not instantiate its own object.
```

Comment:
Typo here. It leads with "The modify" but probably was intended to be "Then, modify"